test/testdata/generate.sh

```diff
@@ -132,16 +135,17 @@ scaffold_test_project project-v2 --project-version=2
 # [Currently, default CLI plugin] - Project version 3 (default) uses plugin go/v3 (default).
 scaffold_test_project project-v3
 scaffold_test_project project-v3-multigroup
-scaffold_test_project project-v3-declarative-v1 --plugins="go/v3,declarative,grafana/v1-alpha"
+scaffold_test_project project-v3-declarative-v1 --plugins="go/v3,declarative,grafana/v1-alpha,monitoring/v1-alpha"
```

Comment:
Could we you please do not add this plugin here? we should have in the declarative one juts the declarative.
Therefore, we need to remove the grafana/v1-alpha from master branch.

Reply:
@camilamacedo86 
We need to test both the `init` and `edit` commands with the new monitoring plugin.
at first I used ` scaffold_test_project project-v4-monitoring --plugins="go/v4-alpha,monitoring.kubebuilder.io/v1-alpha" ` to test the init commend.

but it created a confusing naming under testdata folder 
project-v*-monitoring
project-v*-with-monitoring
 
 Later I followed @varshaprasad96 suggestion:
 "
  I would suggest to test both the init and edit commands. We should have scaffold_test_project project-v4-with-monitoring and the other scaffolding which calls this command chained with other plugins (whichever seems fit), probably `scaffold_test_project project-v4-declarative-v1 --plugins="go/v4-alpha,declarative,grafana/v1-alpha,monitoring/v1-alpha". This will make sure that a user with pre-scaffolded project is able to add the monitoring plugin with the edit command, as well as someone who is scaffolding their project for the first time can chain this plugin with existing ones during init.
  "

Reply:
@varshaprasad96 and @everettraven suggested this to test all the plugins together, in oreder to see there are no conflicts.

Reply:
So thinking about this again (I now think I understand the original problem better than I did before), I understand @camilamacedo86's reasoning for wanting them separate (definitely better for testing/maintenance IMO).

I don't feel particularly strongly one way or another but if I _had_ to choose I would probably vote for separate project scaffolds. I understand that it could be confusing names under the testdata folder but in the end I think we need to do what is better for testing and maintenance. I think maybe naming the projects `project-v*-init-monitoring` and `project-v*-edit-monitoring` might make it a bit more clear which project was created with which subcommand.